wordpress-mobile/WordPress-iOS
# Login: add epilogue screen

Adds the login process's new epilogue screen.

![login-epilogue-first](https://cloud.githubusercontent.com/assets/517257/25297170/6b7ddb28-26a9-11e7-88a2-3f097f98f4dd.gif)

### Notes:
- The epilogue screen itself is currently just a blank screen. I've kept its implementation for another PR.
- The epilogue transition is now a segue, unlike post-post's view stacking solution.
- `Login.storyboard` now uses segues to present each step. As a consequence, each `Signin*` view controller has a subclass named `Login*` with overridden methods to perform the correct segue, etc.
- Changed the feature flag to just `.debug`.
- Magic link logins don't yet get the epilogue screen

To test:
Make sure login still works! All successful logins except magic links should see the epilogue screen:
- wpcom login via user/pass
- wpcom login with 2FA
- self-hosted login

Needs review: @aerych
